### PR TITLE
[stripe] Add support for the Person concept on Stripe Connect Accounts

### PR DESCRIPTION
**Pull request by bruun**
Documentation: https://stripe.com/docs/api/persons?lang=node

The interface overlap a lot with the IIndividual interface, but I have chosen to keep them completely separate to avoid issues going forward if any of them change without the other.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/stripe/index.d.ts [hunk 1]
```diff
@@ -1045,6 +1045,331 @@ declare namespace Stripe {
...
+             */
+            first_name_kana?: string;
+
+            /**
+             * The Kanji variation of the person’s first name (Japan only).
+             */
+            first_name_kanji?: string;
+
+            /**
+             * The person’s gender (International regulations require either “male” or “female”).
```
Comment (pushplay) [inline types/stripe/index.d.ts]: 🤦

File: types/stripe/index.d.ts [hunk 2]
```diff
@@ -1045,6 +1045,331 @@ declare namespace Stripe {
...
+             * The person’s maiden name.
+             */
+            maiden_name?: string;
+
+            /**
+             * Set of key-value pairs that you can attach to an object.
+             * This can be useful for storing additional information about
+             * the object in a structured format.
+             */
+            metadata?: {
```
Comment (pushplay) [inline types/stripe/index.d.ts]: You should pull this out and use `IOptionsMetadata` in create and update and use `IMetadata` in the object.

File: types/stripe/index.d.ts [hunk 3]
```diff
@@ -9421,6 +9746,41 @@ declare namespace Stripe {
                 redirectUrl?: string,
                 response?: IResponseFn<accounts.ILoginLink>,
             ): Promise<accounts.ILoginLink>;
+
```
Comment (pushplay) [inline types/stripe/index.d.ts]: I believe all of these CRUDL functions should have an overload that accepts a `HeaderOptions` param before the `IResponseFn`.

File: types/stripe/stripe-tests.ts [hunk 4]
```diff
@@ -1256,6 +1256,38 @@ stripe.accounts.createLoginLink('acct_17wV8KBoqMA9o2xk', 'http://localhost:3000'
     const created: number = loginLink.created;
     const url: string = loginLink.url;
 });
+
```
Comment (pushplay) [inline types/stripe/stripe-tests.ts]: The account persons tests should probably be another section to make it clear to future contributors.

**Review by pushplay — CHANGES_REQUESTED** [review]
The details look pretty good.  Thanks for the tests.  Just a few small things to take care of.

File: types/stripe/index.d.ts (hunk 1 above)
Comment (bruun) [reply types/stripe/index.d.ts]: Yeah 🤦‍♂️

File: types/stripe/index.d.ts (hunk 3 above)
Comment (bruun) [reply types/stripe/index.d.ts]: Good catch, I didn't think to look at the other CRUDL types and only looked at the documentation where this isn't explicitly mentioned on each function. I _think_ it should be correct now

**Review by pushplay — APPROVED** [review]

**Comment by pushplay** [comment]
Looks good.  Thanks for the changes.

**Pull request merged**